### ISSUE TITLE
man where do I start 

### ISSUE THREAD

**Issue by ApplejuceYT** [Vita3K/Vita3K-Android#233]
man honestly this is just a disappointment or rather to soon of a release I get snapdragon gets the best optimizing of all the chipsets out there but crashes before a boot on Mali device nah that's a hard pill to swallow got a p30 kirin 980 and Mali G76 ps2 runs on 4x and I can't even get a boot in mortal kombat playable version please fix the optimizing for Mali devices I love emulation and especially on Android never thought the vita would be on my phone

**Comment by LHBlitz** [comment]
You can't expect a new emulator that is still no where optimized on PC to just automatically work for an Android phone. This stuff takes time.

**Comment by ApplejuceYT** [comment]
yes I understand that just a shock is all that it crashes in Mali devices
buts works on a mid rage snapdragon phone I'm just angry that Mali if never
gets optimized for anything
